ukrflix/ukrflix-data/src/main/java/org/example/ukrflix/model/Actor.java

```diff
@@ -0,0 +1,92 @@
+package org.example.ukrflix.model;
+
+
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+import org.hibernate.validator.constraints.NotBlank;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.sql.Date;
+import java.util.List;
+import java.util.Set;
+
+@Entity
+public class Actor{
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+    @NotBlank
+    private String firstname;
+    @NotBlank
+    private String lastname;
+    @NotNull
+    private Date birthday;
```

Comment:
let's use java.time.LocalDate instead of java.sql.Date
To read
https://howtodoinjava.com/java/date-time/localtime-to-sql-time/
https://thorben-janssen.com/hibernate-5-date-and-time/